azurebatchdockerhelp.md

```diff
@@ -0,0 +1,56 @@
+## Azure Batch with docker containers
+Azure Batch has the functionality to run jobs and tasks within a docker container. Using docker containers to execute REEF jobs has the benefit of setting up the dependencies in light-weight docker containers instead of the vm node. This section describes how you can configure the pool and REEF to execute REEF jobs inside docker containers.
+
+####	1. Create Dockerfile for your OS with REEF and other dependencies 
+##### Option: Windows Docker image
+
+Following example of dockerfile targets the windowsservercode on dockerhub and the pre-requisites for java and other path variables (the reference to vcruntime140.dll is since the default Windows Server core image is missing this important dll to be able to run java programs):
```

Comment:
>he reference to vcruntime140.dll is since the default Windows Server core image is missing this important dll to be able to run java programs [](start = 134, length = 141)

It might be worth putting these notes as comments directly in the Dockerfile